build.sc

```diff
@@ -233,12 +233,14 @@ object `llvm-firtool` extends JavaModule with ChipsAlliancePublishModule {
 // ******************** WARNING ********************
 // This is extremely manual and changing dependencies IN ANY WAY (including bumping version)
 // requires carefully checking the packages to shade and dynamic ivy deps in the outer project
-object `firtool-resolver` extends Cross[FirtoolResolver]("2.13", "2.12")
-trait FirtoolResolver extends CrossScalaModule with ChipsAlliancePublishModule { root =>
+object `firtool-resolver` extends Cross[FirtoolResolver]("2.13", "2.12", "3.4", "3.3")
+trait FirtoolResolver extends CrossSbtModule with ChipsAlliancePublishModule { root =>
 
   override def crossScalaVersion = Map(
     "2.13" -> "2.13.11",
-    "2.12" -> "2.12.18"
+    "2.12" -> "2.12.18",
+    "3.4" -> "3.4.2",
+    "3.3" -> "3.3.3"
```

Comment:
```suggestion
    "3" -> "3.3.3"
```
Scala 3 maintains binary compatibility across all minor versions so there is no reason to cross compile for any version but 1 in Scala 3 (unlike Scala 2 where it you do have to compile differently for 2.12 and 2.13.